ui/components/multichain/account-overview/account-overview-layout.tsx

```diff
@@ -12,10 +37,73 @@ export const AccountOverviewLayout = ({
   children,
   ...tabsProps
 }: AccountOverviewLayoutProps) => {
+  const dispatch = useDispatch();
+  const slides = useSelector(getSlides);
+  const totalBalance = useSelector(getSelectedAccountCachedBalance);
+  const isLoading = useSelector(getAppIsLoading);
+
+  const hasZeroBalance = totalBalance === '0x00';
+
+  ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
+  const location = useLocation();
+  const { openBridgeExperience } = useBridging();
+  const defaultSwapsToken = useSelector(getSwapsDefaultToken);
+  ///: END:ONLY_INCLUDE_IF
+
+  useEffect(() => {
+    ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
+    const handleBridgeClick = () => {
+      if (defaultSwapsToken) {
+        openBridgeExperience(
+          'Home',
+          defaultSwapsToken,
+          location.pathname.includes('asset') ? '&token=native' : '',
+        );
+      }
+    };
+    ///: END:ONLY_INCLUDE_IF
+
+    const fundSlide = {
+      ...FUND_SLIDE,
+      undismissable: hasZeroBalance,
+    };
+
+    const defaultSlides = [
```

Comment:
Do we need default slide in useEffect?

Reply:
Why not? How would you implement this otherwise?

Reply:
Ah my bad, the onClick is defined here as well. you are right